cocos/2d/CCRenderTexture.cpp

```diff
@@ -252,16 +259,49 @@ bool RenderTexture::initWithWidthAndHeight(int w, int h, Texture2D::PixelFormat
 
         if (depthStencilFormat != 0)
         {
-            //create and attach depth buffer
-            glGenRenderbuffers(1, &_depthRenderBufffer);
-            glBindRenderbuffer(GL_RENDERBUFFER, _depthRenderBufffer);
-            glRenderbufferStorage(GL_RENDERBUFFER, depthStencilFormat, (GLsizei)powW, (GLsizei)powH);
-            glFramebufferRenderbuffer(GL_FRAMEBUFFER, GL_DEPTH_ATTACHMENT, GL_RENDERBUFFER, _depthRenderBufffer);
-
-            // if depth format is the one with stencil part, bind same render buffer as stencil attachment
-            if (depthStencilFormat == GL_DEPTH24_STENCIL8)
+            
+            
+            const char* extString = (const char*)glGetString(GL_EXTENSIONS);
```

Comment:
This is no need to query the extension every time a RenderTexture created, you should put it in the `Configuration` class.